pathfinder/tests/test_mocked_integration.py

```diff
@@ -20,7 +20,7 @@ def test_pfs_with_mocked_events(
     token_networks: List[TokenNetwork],  # just used for addresses
     addresses: List[Address],
     pathfinding_service_mocked_listeners: PathfindingService,
-    channel_descriptions: List
+    channel_descriptions_case_1: List
 ):
```

Comment:
We could potentially make this topology independent, i.e. together with a given topology, hand over the expected results that are to be asserted by the test.

Reply:
I tried this, but right now the tests seem to different for this to make sense. Some test routing, some test routing, some routing with fees, some routing with fees and changes bias. 
We can definitively improve stuff, but lets do that in another PR: